### DISCUSSION TITLE
Getting “you’re running low on credits” email even though I have auto-reload enabled

### DISCUSSION THREAD

**Discussion by KaidenR** [deepgram/community#593] [General help] [answered]
The email says "To avoid this message in the future you can enable Auto-reload to automatically refresh 
your balance when it's low." But I DO have it enabled.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment]
@KaidenR  are you still having an issue with this?

**Reply by KaidenR** [reply]
Looks like I haven't gotten the email since January so it may be fixed.

**Comment by jpvajda** [comment, marked as answer]
@KaidenR  ok thanks for letting us know. We recently did some updates to our transactional emails so it's possible we resolved a bug on our end that might have been sending these emails incorrectly in certain cases. 

Glad to hear it's resolved.
